src/flow-spec/IFlow.ts

```diff
@@ -38,7 +37,6 @@ export interface IFlow {
   languages: ILanguage[]
   blocks: IBlock[]
 
-  firstBlockId: string
```

Comment:
Hi @bulatgab @AlteaDown, can you help me understand the story behind this? I believe we'd rather want to update the server to provide `IFlow::firstBlockId` and `IBlockExit::config` for us. 

I can definitely understand removing orgId though! :)

Reply:
I don't have preferences for this (I agree on changing the server response). @AlteaDown ?

Reply:
@bzabos The primary reason this is here, is that it is not in FLOIP spec, and we haven't even realized it was missing until now, when we added validation tests to Clipboard, so that must mean we don't use it. Do you think it should be here and added to FLOIP spec?

Reply:
Three tasks here:
- [x] add to proposed flow spec updates
- [ ] update flow runner to leverage this property
- [ ] update backend to send this property